Remove releases description from the Russian part

likhobory opened the pull request:
- Remove releases description from the Russian part 
- Additional compression for some images 
- Add blog posts "AS IS" to the Russian part as described in https://github.com/salesagility/SuiteDocs/issues/46#issuecomment-376250153

pgorod commented:
Hi @likhobory, I just updated the Russian branch on our repo and I've been looking at this PR. 

I see some changes on the English versions of the Release files, are these intended? I don't think you meant to include those files in your commit.

```
content/admin/Roadmap.adoc 
content/admin/releases/7.10.x/_index.en.adoc
content/admin/releases/7.11.x/_index.en.adoc
```

and also **some image files** in the English and Spanish versions. 

If I am correct, can you please clean up the PR so I can merge it? Or tell me if there's anything you need me to do. Thanks!

likhobory commented:
> content/admin/Roadmap.adoc 
> content/admin/releases/7.10.x/_index.en.adoc
> content/admin/releases/7.11.x/_index.en.adoc

Hi Pedro,
you're right, I removed them by mistake, fixed.


> some image files in the English and Spanish versions

I have compressed various images, regardless of their version, guided only by their size. Their quality after compression was not affected, so if you don't mind, I would like to keep all these images in this PR  (btw, some images from the  static/images/en/community folder are simultaneously used in both English and Russian docs).

pgorod commented:
Ok, I don't have time to check all those images so I will just trust you on this one... ;-)
